src/Pagination/Footer/Buttons/Buttons.js

```diff
@@ -46,7 +47,7 @@ const Buttons = ({
 
   return (
     <>
-      {captcha !== undefined && currentPage === pages.length - 1 && (
+      {!isSingleField && captcha !== undefined && currentPage === pages.length - 1 && (
```

Comment:
isSingleField is useless here cause this current condition is fully covers all stuff related condition